lib/geokit/geocoders.rb

```diff
@@ -207,9 +206,9 @@ def self.transcode_to_utf8(body)
     # -------------------------------------------------------------------------------------------
     # "Regular" Address geocoders
     # -------------------------------------------------------------------------------------------
-    require File.join(File.dirname(__FILE__), 'geocoders/base_ip')
-    Dir[File.join(File.dirname(__FILE__), '/geocoders/*.rb')].each {|f| require f}
+    require File.join(File.dirname(__FILE__), "geocoders/base_ip")
+    Dir[File.join(File.dirname(__FILE__), "/geocoders/*.rb")].each {|f| require f}
```

Comment:
Line is too long. [82/80]<br>Space between { and | missing.<br>Space missing inside }.